File: python/e_1-50/e_6-10/e_9_spec_pythag.py
import math as m

n = 1000

end = False
for a in range(1, n):
    for b in range(a, n):
        c = m.sqrt(a**2 + b**2)
        if a + b + c == n:
            print(int(a*b*c))
            end = True
            break
    if end:
        break

File: typescript/e_1-50/e_6-10/e_9_spec_pythag.ts
var n: number = 1000
var end: boolean = false
var c: number = 0

for (var a = 1; a < n; a++) {
    for (var b = 1; b < n; b++) {
        c = Math.sqrt(a**2 + b**2)
        if (a + b + c === n) {
            console.log(a*b*c)
            end = true
            break
        }
    }
    if (end)    break
}
